doc/UsersGuide/source/InputsOutputs.rst

```diff
@@ -4,6 +4,29 @@
 Inputs and Outputs
 *************************
 
+The ufs-weather-model can be configured as one of several applications, from a single component model atmosphere model to a fully coupled model with multiple earth system components (atmosphere, ocean, sea-ice and mediator). Currently the supported configurations are:
```

Comment:
Change "from a single component model atmosphere model" to "from a single component atmosphere model"

Reply:
Or to "from a single component model with atmosphere component"